src/useAsyncAbortable/useAsyncAbortable.ts

```diff
@@ -1,23 +1,23 @@
 import { useMemo, useRef } from 'react';
-import { useAsync, UseAsyncMeta, UseAsyncActions, AsyncState } from '../useAsync/useAsync';
+import { AsyncState, useAsync, UseAsyncActions, UseAsyncMeta } from '../useAsync/useAsync';
 
 export interface UseAsyncAbortableActions<Result, Args extends unknown[] = unknown[]>
   extends UseAsyncActions<Result, Args> {
   /**
-   *  Abort currently running async.
+   *  Abort the currently running async function invocation.
    */
   abort: () => void;
 
   /**
-   * Abort currently running async and reset state to initial, when async function haven't been executed.
+   * Reset state to initial, when the async function hadn't been executed.
```

Comment:
I think it's important to note that `reset()` seems to also run `abort()` under the hood, thus the wording of the original, I think.